build-support/bin/native/bootstrap_rust.sh

```diff
@@ -75,7 +75,7 @@ function bootstrap_rust() {
         fi
       done
 
-      ln -fs "$(dirname "${cargo_bin_dir}")/lib/rustlib/src/rust/src"
+      ln -fs "$(dirname "${cargo_bin_dir}")/lib/rustlib/src/rust/src" .
```

Comment:
`ln` defaults implicitly to using `.`. This makes it explicit.

Reply:
What does this create in the current directory? Is it a symlink named `src`? I really prefer making it explicit what gets created by using the `ln TARGET LINK_NAME` form instead of the `.`. I may be confused.

Reply:
I honestly do not know :/ I could not figure out how to fix this one so just went with what shellcheck said is easiest to keep current behavior.

Refer to https://github.com/koalaman/shellcheck/wiki/SC2226.

Reply:
This creates a link in the current directory called `src`. I'm happy with any of no second argument, `.`, or `src`.

Reply:
I would personally prefer using `src`, but that's just because I personally haven't seen the `.` format very often. However, it appears to be very standardized and understandable enough, so making the minimum change here (using `.`) seems to make sense.

Reply:
It would seem so:
```
$ cd path/to/pants
$ ln -fs src/python/ .
$ ls | grep python
python
```

```suggestion
      ln -fs "$(dirname "${cargo_bin_dir}")/lib/rustlib/src/rust/src" "src"
```

Reply:
Wonderful, I like `src` much more. Thanks!